Content.Shared/SimpleStation14/Species/Shadekin/Systems/ShadekinSystem.cs

```diff
@@ -0,0 +1,120 @@
+using Content.Shared.Examine;
+using Content.Shared.SimpleStation14.Species.Shadekin.Components;
+using Robust.Shared.Network;
+using Content.Shared.IdentityManagement;
+using Content.Shared.SimpleStation14.Species.Shadekin.Events;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared.SimpleStation14.Species.Shadekin.Systems
+{
+    public sealed class ShadekinSystem : EntitySystem
+    {
+        [Dependency] private readonly ShadekinPowerSystem _powerSystem = default!;
+        [Dependency] private readonly INetManager _net = default!;
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<ShadekinComponent, ExaminedEvent>(OnExamine);
+            SubscribeLocalEvent<ShadekinComponent, ComponentInit>(OnInit);
+
+            SubscribeLocalEvent<ShadekinComponent, ComponentGetState>(GetCompState);
+            SubscribeLocalEvent<ShadekinComponent, ComponentHandleState>(HandleCompState);
+
+            // Due to duplicate subscriptions, removal of the alert is in ShadekinDarkenSystem
+        }
+
+        private void OnExamine(EntityUid uid, ShadekinComponent component, ExaminedEvent args)
+        {
+            if (args.IsInDetailsRange && !_net.IsClient)
```

Comment:
If you can get away with
```suggestion
            if (args.IsInDetailsRange)
```
please do. Prediction is 💋

Reply:
seemingly has no effect